test/screenshot/golden.json

```diff
@@ -574,6 +582,14 @@
       "desktop_windows_ie@11": "https://storage.googleapis.com/mdc-web-screenshot-tests/advorak/2018/09/26/06_08_27_877/spec/mdc-drawer/classes/modal.html.windows_ie_11.png"
     }
   },
+  "spec/mdc-drawer/classes/permanant-below-top-app-bar.html": {
```

Comment:
Since you've removed this from screenshot tests, remove these from golden.json too.